capstone/src/main/java/com/google/sps/servlets/DeletePollOptionServlet.java

```diff
@@ -50,34 +47,36 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
   }
 
-  /**
-   * Sets the variables maxVotes and maxVotesId by iterating through the entities.
-   *
-   * @param results queried results
-   * @return maxVotesId the id of option with maximum number of votes
-   */
-  private long getMaxVotedId(PreparedQuery results) {
-    long maxVotedId = 0;
-    int maxVotes = 0;
-    Iterable<Entity> resultsIterable = results.asIterable();
-    // Set maxVotedId to be first one in iterable of Entities
-    if (Iterables.size(resultsIterable) > 0) {
-      maxVotedId = results.asIterable().iterator().next().getKey().getId();
+  private List<Long> getOptionsList(
+      HttpServletRequest request, HttpServletResponse response, DatastoreService datastore)
+      throws IOException {
+    Group group = new Group(null, null, null, null, "", "", 0);
+    Entity entity = group.getGroupEntity(request, response, datastore);
```

Comment:
Since the getGroupEntity method is static, can you not just call Group.getGroupEntity instead of creating this dummy group that isn't used?